sys/include/net/netopt.h

```diff
@@ -269,6 +269,114 @@ typedef enum {
      */
     NETOPT_LAST_ED_LEVEL,
 
+    /**
+     * @brief   Get/Set preamble length as uint16_t in host byte order.
+     */
+    NETOPT_PREAMBLE_LENGTH,
+
+    /**
+     * @brief   Enable/disable CRC check
+     */
+    NETOPT_CRC,
```

Comment:
Maybe`NETOPT_INTEGRITY_CHECK`? Not all protocols use CRC for that and this way we can use it for all those, too. Doc needs adaption, too. E.g. "Enable/disable checksum check".